src/controllers/keyboard/keyboardeventfilter.cpp

```diff
@@ -156,15 +180,139 @@ QKeySequence KeyboardEventFilter::getKeySeq(QKeyEvent* e) {
 #endif
 }
 
-void KeyboardEventFilter::setKeyboardConfig(ConfigObject<ConfigValueKbd>* pKbdConfigObject) {
+void KeyboardEventFilter::setEnabled(bool enabled) {
+    m_enabled = enabled;
+    m_pConfig->set(ConfigKey("[Keyboard]", "Enabled"), ConfigValue(enabled));
```

Comment:
nit here, but perhaps use the templated version since you already use `getValue` later?
```suggestion
    m_pConfig->setValue(ConfigKey("[Keyboard]", "Enabled"), enabled);
```

Reply:
Thanks, done.